lms/djangoapps/discussion/tests/test_tasks.py

```diff
@@ -214,6 +214,8 @@ def test_send_discussion_email_notification(self, user_subscribed):
             self.assertEqual(expected_recipient, actual_message.recipient)
             self.assertEqual(self.course.language, actual_message.language)
             self._assert_rendered_email(actual_message)
+            with mock.patch('analytics.track') as mock_analytics_track:
+                self.assertEqual(mock_analytics_track.call_count, 1)
```

Comment:
Do you want to make some assertion(s) about what `mock_analytics_track` is called with?